video_creation/final_video.py

```diff
@@ -139,26 +140,26 @@ def make_final_video(
     #    # lowered_audio = audio_background.multiply_volume( # todo get this to work
     #    #    VOLUME_MULTIPLIER)  # lower volume by background_audio_volume, use with fx
     #    final.set_audio(final_audio)
-    final = Video(final).add_watermark(
+    '''final = Video(final).add_watermark(
         text=f"Background credit: {background_config[2]}", opacity=0.4
-    )
+    )''' #  todo add watermark
```

Comment:
You got to finish those todo's...

Reply:
You're right, I forgot it because it was difficult to do. With the commit I just did it is added.